File: mean/service/models/eqiDetail.js
var mongoose = require('mongoose');

module.exports = mongoose.model('EqiRDetail',{
                                stateCode : String,
                                stateDescription : String,
                                countyCode : String,
                                countyDescription : String,
                                variableCode : String,
                                variableDescription : String,
                                variableValue : Number,
                                domain : String
                });

File: roar/app/models/eqi_detail.rb
class EqiDetail
  include Mongoid::Document
  field :stateCode, type: String
  field :stateDescription, type: String
  field :countyCode, type: String
  field :countyDescription, type: String
  field :variableCode, type: String
  field :variableDescription, type: String
  field :variableValue, type: Float
  field :domain, type: String
end
